Species/Humanity.md

```diff
@@ -66,7 +66,7 @@ This is a centralized structure of what happened to humanity in the last hundred
 
-* 2298-2310 : Robotics becomes the hot new technology in the millennium of the 24th century, as increasingly more intelligent and sophisticated expert systems and automated task robots show up, primarily in industrial scale mining, logistics handling (especially in the field of drop pods flow) and construction projects, as well in manufacturing and assembling. Less raw manpower is required and many people lose their jobs, leading to internal unrest as unemployment skyrockets. On a lighter note, cybernetics follows the rapid pace of robotics and IT technology, soon providing the first truly viable and rejection-free cyberlimbs and organ assistant systems for a reasonable price, as well the “MMI”, the possibility of living inside a fully robotic shell, although the technology is relatively new and only truly represented in the fringes of human space, where hostile and dangerous environments are commonplace. Under the guise of Zheng-Hu Robotics, many a miner finally has all his limbs together. Later in time, Zheng-Hu abandons robotics and further specializes in medical application of mechanical parts in the human body, before completely transforming into the biggest human pharmaceutics corporation, as they capitalize anti-gravitation sickness compounds.
+* 2298-2310 : Robotics becomes the hot new technology in the millennium of the 24th century, as increasingly more intelligent and sophisticated expert systems and automated task robots show up, primarily in industrial scale mining, logistics handling (especially in the field of drop pods flow) and construction projects, as well in manufacturing and assembling. Less raw manpower is required and many people lose their jobs, leading to internal unrest as unemployment skyrockets. On a lighter note, cybernetics follows the rapid pace of robotics and IT technology, soon providing the first truly viable and rejection-free cyberlimbs and organ assistant systems for a reasonable price, as well the “MMI”, the possibility of living inside a fully robotic shell, although the technology is relatively new and only truly represented in the fringes of human space, where hostile and dangerous environments are commonplace. Under the guise of Zeng-Hu Robotics, many a miner finally has all his limbs together. Later in time, Zeng-Hu abandons robotics and further specializes in medical application of mechanical parts in the human body, before completely transforming into the biggest human pharmaceutics corporation, as they capitalize anti-gravitation sickness compounds.
```

Comment:
*in the 24th century,

*sophisticated technology and automated

*industrial-scale

*"Mind-Machine Interface"

*Wearing Zeng-Hu Robotics-branded limbs, formerly crippled miners return to work, and wounded veterans can walk again.

*As it grows, Zeng-Hu abandons